src/util/hass-util.html

```diff
@@ -236,6 +236,8 @@
       return activated ? 'mdi:thermometer' : 'mdi:fire';
     case 'light':
       return activated ? 'mdi:brightness-5' : 'mdi:brightness-7';
+    case 'lock':
+      return activated ? 'mdi:lock-open' : 'mdi:lock';
```

Comment:
Invert this. The lock should be closed in the normal state and open in the alert state. See window eg.

Reply:
Agree it could go either way, as the names are rather vague. But the majority of the classes seem to have their name as the alert state (e.g. 'cold', 'connectivity', 'heat', 'light'), so that's what I went with.